src/PKSim.Core/Model/ParameterExtensions.cs

```diff
@@ -127,5 +128,17 @@ public static bool CanBeDefinedAsAdvanced(this IParameter parameter)
 
          return parameter.CanBeVariedInPopulation && !parameter.IsChangedByCreateIndividual;
       }
+
+      public static IEnumerable<IParameter> AllRelatedRelativeExpressions(this IParameter relativeExpressionParameter)
```

Comment:
Should we return an IReadOnlyList? (e.g. TOList at the end). This will always be used with a ToList probably on the caller side otherwise